src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1964,7 +1957,7 @@ class A
             ]]></Document>)
                 state.SendTypeChars("X")
                 Await state.AssertCompletionSession()
-                Assert.False(state.CurrentCompletionPresenterSession.CompletionItems.Any(Function(i) i.DisplayText = "X"))
+                Assert.True(state.CompletionItemsContainsAll({"X"}))
```

Comment:
❕ This appears to be the opposite of the original assertion

Reply:
You are right. This is covered in [https://github.com/dotnet/roslyn/pull/28219](https://github.com/dotnet/roslyn/pull/28219)

Thank you!

---
In reply to: [207999785](https://github.com/dotnet/roslyn/pull/29016#discussion_r207999785) [](ancestors = 207999785)